sds_data_manager/lambda_code/batch_starter.py

```diff
@@ -338,6 +336,60 @@ def extract_components(filename: str):
     return components
 
 
+def send_lambda_put_event(command_parameters):
+    """Sends custom PutEvent to EventBridge
```

Comment:
So glad you have an example here. My question is: I thought that this was the information that would be used to populate the StatusTracking table, which has the following columns. 

    id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
    file_to_create_path = Column(String, nullable=False)
    status = Column(statuses, nullable=False)
    job_definition = Column(String, nullable=False)
    ingestion_date = Column(DateTime, nullable=True)
    
We've had a lot of conversations so maybe that's changed. You might just need to get me up to speed on this.

Reply:
That's true. This event is suppose to send custom putevent from batch starter to indexer lambda to populate status table. But we send this information to initialized that table with `file_to_create`, `status` and initialize others a null. Then when batch job finishes we update table with new information and update status as well based on batch job event.